fizzBuzz/fizzBuzz.ts

```diff
@@ -0,0 +1,8 @@
+const fizzBuzz = (num: number) => {
+  for (let i = 1; i < num; i++) {
+    const str = (i % 3 ? '' : 'Fizz') + (i % 5 ? '' : 'Buzz');
```

Comment:
Much better use if statement in such cases.

Reply:
done